perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -706,6 +726,6 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 
 	<ldap:ldap-template />
 
-	<import resource="file:${perun.conf.custom}/perun-ldapc-attributes.xml" />
+	<import resource="file:${perun.conf.custom}/perun-ldapc-*.xml" />
```

Comment:
Are there any other expected file names ? Because we have empty ``perun-ldapc-attributes.xml`` present in our deployment template. I believe we can still use it to re-define all attribute configurers.